great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -145,14 +145,19 @@ class ExpectColumnQuantileValuesToBeBetween(ColumnExpectation):
 
     def validate_configuration(self, configuration: Optional[ExpectationConfiguration]):
         super().validate_configuration(configuration)
-
         try:
             assert (
                 "quantile_ranges" in configuration.kwargs
-            ), "quantile ranges must be provided"
+            ), "quantile_ranges must be provided"
             assert isinstance(
                 configuration.kwargs["quantile_ranges"], dict
             ), "quantile_ranges should be a dictionary"
+            assert all(
+                [
+                    True if None in x or x == sorted(x) else False
+                    for x in configuration.kwargs["quantile_ranges"]["value_ranges"]
+                ]
+            ), "quantile_ranges must consist of ordered pairs"
```

Comment:
```suggestion
    for value_range in configuration.kwargs["quantile_ranges"]["value_ranges"]:
        assert (None in value_range) or (value_range == sorted(value_range)), "quantile_ranges must consist of ordered pairs"
```

Thoughts on unwrapping this comprehension for readability? It's less performant due to a higher number of asserts but I think it's a viable alternative. I trust your decision here I'm happy either way 😄

Reply:
For an expectation already prone to bloat on large datasets, I'm inclined to favor something even marginally more performant, but happy to reassess if desired in the future!